dev/io.openliberty.data.internal_fat/test-applications/DataTestApp/src/test/jakarta/data/web/DataTestServlet.java

```diff
@@ -4953,43 +4953,51 @@ public void testReservedKeywordsInEntityPropertyNames() {
                                              .map(o -> o.a)
                                              .collect(Collectors.toList()));
 
-        // "Or" in middle of entity property name is possible to to use of @Query.
+        // "Or" in middle of entity attribute name is possible to to use of @Query.
```

Comment:
Nit: remove one of the "to"

Reply:
> Nit: remove one of the "to"

That's extra credit for spotting a typo in the code comments. I added a commit to remove it.